src/IO/FileExplorer.h

```diff
@@ -1,11 +1,10 @@
 #pragma once
 
-#include <string>
-#include <vector>
-#include <regex>
-
-#include "../helpers/ErrorReport.h"
-
+#include <vector> //vector
+#include <regex> // regex
```

Comment:
Why the comments?

Reply:
Ah, var för min egen skull för att hålla koll på vilka filer jag gått igenom osv.. kan ta bort!